File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/TerminPostVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class TerminPostVM
    {
        public int TerminID { get; set; }
        public string Datum { get; set; }
        public string Vrijeme { get; set; }
        public string Napomena { get; set; }
        public bool? IsOdobren { get; set; }
        public bool IsEvidentiran { get; set; }
        public bool IsDeleted { get; set; }

        public string ImePrezime { get; set; }
        public int PacijentID { get; set; }
        public string Usluga { get; set; }
    }
}

File: app/src/main/java/com/example/ordinacija/Data/TerminPostVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;
import java.util.Date;

public class TerminPostVM implements Serializable {
    public int TerminID;
    public String Datum;
    public String Vrijeme;
    public String Napomena;
    public boolean IsOdobren;
    public boolean IsEvidentiran;
    public boolean IsDeleted;
    public String ImePrezime;
    public int PacijentID;
    public String Usluga;

    public TerminPostVM(String Datum,String Vrijeme,String Opis,int PacijentID,String  UslugaID){
        this.Datum=Datum;
        this.Vrijeme=Vrijeme;
        this.Napomena=Opis;
        this.PacijentID=PacijentID;
        this.Usluga=UslugaID;
    }
}
